tinygrad/llops/ops_gpu.py

```diff
@@ -146,12 +151,11 @@ def _ast(x: Union[GPUBuffer, LazyOp], buf_names: Dict[GPUBuffer, str], code_for_
       if not allow_reduce and type(x.op) in [ProcessingOps, ReduceOps]:
         return "acc"
       srcs_code = [_ast(src, buf_names, code_for_op) for src in x.src]
-      code = code_for_op[x.op]
       if len(srcs_code) >= 1:
-        code = code.replace("A", srcs_code[0])
+        return code_for_op[x.op].replace("A", srcs_code[0])
       if len(srcs_code) >= 2:
-        code = code.replace("B", srcs_code[1])
-      return code
+        return code_for_op[x.op].replace("B", srcs_code[1])
```

Comment:
Not right, you have to replace A also

Reply:
Ah I see. I misunderstood the control flow while refactoring. I'll fix it